distributions/otelcol-otlp/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.19 as certs
+RUN apk --update add ca-certificates
+
+FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --chmod=755 otelcol-otlp /otelcol-otlp
+ENTRYPOINT ["/otelcol-otlp"]
+CMD ["--config", "/etc/otelcol-otlp/config.yaml"]
```

Comment:
For the k8s distribution we [stopped including a default config](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/507#discussion_r1541420891).  I believe the some logic applies here.

Reply:
I didn't include a default config file, but I kept the default location, as I thought it wouldn't have those problems, and it would avoid an extra step during step. Should I remove it?

Reply:
I feel like adding this default path in the systemd service definition makes sense, but here we can just remove the `CMD` part and force users to provide their own command line arguments